### ISSUE TITLE
How to get LayoutLM softmax output?

### ISSUE THREAD

**Issue by Miguelnns** [deepdoctection/deepdoctection#363]
Hi, when running LayoutLM for document classification, how can I get the softmax output with the probability of each class, instead of just the name of the class with higher probability? This is the code i'm currently using:

file_path = 'path/to/file'
doc_classifier = get_layoutlm_pipeline()
df = doc_classifier.analyze(path=file_path)
df.reset_state()

doc = iter(df)
page = next(doc)
print("Document type:",page.document_type.value)

**Comment by JaMe76** [comment]
Neither logits nor softmax are being saved temporarily in any class and you therefore need to modify the code slightly by adding a softmax attribute to the LayoutLM model wrapper and save it temporarily while processing your image.

The softmax for sequence classification is being calculated here:

https://github.com/deepdoctection/deepdoctection/blob/10599409663eb3331f60f0395c9a9d6b8d23754e/deepdoctection/extern/hflayoutlm.py#L238
